### PR TITLE
added gamerherz

### PR DESCRIPTION
**Pull request by Nicouschulas**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/gamerherz/background@2x.png
[the remark this answers is not in the document]
Comment (Nicouschulas) [reply minecraft_servers/gamerherz/background@2x.png:1]: Yeah, but a there is a bug. I asked some others evertime we upload it the first time it is false comressd. Any idea how to fix this?

File: minecraft_servers/gamerherz/background.png
[the remark this answers is not in the document]
Comment (DoJapHD) [reply minecraft_servers/gamerherz/background.png:1]: Still not compressed
Let's try this:
Remove the files completely with a commit and then readd the compressed files with a commit.

File: minecraft_servers/gamerherz/background@2x.png
[the remark this answers is not in the document]
Comment (DoJapHD) [reply minecraft_servers/gamerherz/background@2x.png:1]: Aswell

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/gamerherz/background.png
[the remark this answers is not in the document]
Comment (Nicouschulas) [reply minecraft_servers/gamerherz/background.png:1]: ok I try this

**Comment by DoJapHD** [comment]
Upload these two again
https://workupload.com/archive/nWmBh3FcAx

**Comment by Nicouschulas** [comment]
> Upload these two again https://workupload.com/archive/nWmBh3FcAx

ok I try it

**Review by DoJapHD — APPROVED** [review]
There we go...

**Comment by Nicouschulas** [comment]
> There we go...

Hmm strange. But thanks

**Pull request merged**